src/lib/__tests__/format.test.js

```diff
@@ -0,0 +1,136 @@
+import {
+    formatNumber,
+    addDistanceLabel,
+    digitGroupSeparator,
+    formatBytes,
+    humanize,
+    pascalToCamelCase,
+    formatLatLong,
+    formatCelsiusTemperature,
+    humanizeCamelCase,
+} from '../format';
+
+describe('format', () => {
+    const units = {
+        speedUnit: 'mph',
+        singleUnit: 'mile',
+        distanceUnit: 'miles',
+        distanceAbbrvUnit: 'mi'
+    };
+
+    describe('addDistanceLabel()', () => {
+        it('will add the word mile to distance 1', () => {
+            const dist = 1;
+            const output = addDistanceLabel(dist, units);
+            expect(output).toBe('1 mi');
+        });
+
+        it('will add the word miles to distance other than 1', () => {
+            const dist = 12;
+            const output = addDistanceLabel(dist, units);
+            expect(output).toBe('12 mi');
+        });
+    });
+
+    test('digitGroupSeparator', () => {
```

Comment:
question: was there a reason to use "test" instead of "it" tests?

Reply:
i updated them to use it now, thanks for catching.